src/main/scala/com/github/takezoe/solr/scala/QueryBuilderBase.scala

```diff
@@ -281,4 +281,37 @@ trait QueryBuilderBase[Repr <: QueryBuilderBase[Repr]] {
     )
   }
 
+
+  /**
+   * Added in response to #63 Missing spatial parameters while building solr query
+   * Sets Spatial field called pt.
+   * @param value : value for point , typically represent lat,long
+   */
+  def spatialPt(value: String): Repr = {
```

Comment:
`def spatialPt(lat: Double, lon: Double)` might be a better interface. What do you think?

Reply:
i agree . Raising new pull request